### PR TITLE
Notification content type branch

### PR DESCRIPTION
**Pull request by Jumaodhiss**
We are introducing a new property 'notificationContentType' in subscription creation payload for supporting multiple notification content types through MS Graph Webhooks notifications.
The current  Graph Webhooks notifications are delivered with _application/json_ content-type. This property will therefore ensure that subscriptions can be configured to deliver other content-types e.g. _application/jwt, application/xml_ etc.

### CODE REVIEW

File: api-reference/beta/includes/snippets/objc/create-subscription-from-subscriptions-objc-snippets.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ MSGraphSubscription *subscription = [[MSGraphSubscription alloc] init];
 [subscription setExpirationDateTime: "2016-11-20T18:23:45.9356913Z"];
 [subscription setClientState:@"secretClientValue"];
 [subscription setLatestSupportedTlsVersion:@"v1_2"];
+[subscription setNotificationContentType:"application/json"];
```
Comment (baywet) [inline api-reference/beta/includes/snippets/objc/create-subscription-from-subscriptions-objc-snippets.md:21]: @Jumaodhiss no need to update the code sample, just the http snippets. The code samples are generated autoamtically from the http snippets

File: api-reference/beta/api/subscription-get.md [hunk 2]
```diff
@@ -158,7 +158,8 @@ Content-length: 252
   "latestSupportedTlsVersion": "v1_2",
   "encryptionCertificate": "",
   "encryptionCertificateId": "",
-  "includeResourceData": false
+  "includeResourceData": false,
+  "notificationContentType": "application/json"
```
Comment (baywet) [inline api-reference/beta/api/subscription-get.md:162]: @Jumaodhiss I can't find any reference of this property in the schema. But I'm guessing you're putting this PR together for the API review?

**Review by baywet — CHANGES_REQUESTED** [review]
On top of the comments I made, the changelog entry is missing

File: api-reference/beta/api/subscription-get.md (hunk 2 above)
Comment (Jumaodhiss) [reply api-reference/beta/api/subscription-get.md:162]: Yes this is for the API review

Comment (Jumaodhiss) [reply api-reference/beta/api/subscription-get.md:162]: @baywet The API review is already completed. This is required for publishing the schema

File: api-reference/beta/includes/snippets/objc/create-subscription-from-subscriptions-objc-snippets.md (hunk 1 above)
Comment (Jumaodhiss) [reply api-reference/beta/includes/snippets/objc/create-subscription-from-subscriptions-objc-snippets.md:21]: Thanks @baywet. I wasn't aware. Should we remove this change and let it be generated automatically

**Comment by Jumaodhiss** [comment]
I've added the entries into the changelog as well

**Review by baywet — APPROVED** [review]

File: api-reference/beta/includes/snippets/objc/create-subscription-from-subscriptions-objc-snippets.md [hunk 3]
```diff
@@ -18,6 +18,7 @@ MSGraphSubscription *subscription = [[MSGraphSubscription alloc] init];
 [subscription setExpirationDateTime: "2016-11-20T18:23:45.9356913Z"];
```
Comment (davidmu1) [inline api-reference/beta/includes/snippets/objc/create-subscription-from-subscriptions-objc-snippets.md:18]: The code snippets should not be changed. They are automatically generated on a weekly bases.

**Review by davidmu1 — APPROVED** [review]

**Pull request closed without merging**